### PR TITLE
Initial commit for android

### PR DESCRIPTION
**Pull request by yuki24**
See #730

device-2017-08-28-150524

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+suppress_comment=\\(.\\|\n\\)*\\$FlowFixMe\\($\\|[^(]\\|(\\(>=0\\.\\(4[0-9]\\|[1-3][0-9]\\|[0-9]\\).[0-9]\\)? *\\(site=[a-z,_]*react_native[a-z,_]*\\)?)\\)
+suppress_comment=\\(.\\|\n\\)*\\$FlowIssue\\((\\(>=0\\.\\(4[0-9]\\|[1-3][0-9]\\|[0-9]\\).[0-9]\\)? *\\(site=[a-z,_]*react_native[a-z,_]*\\)?)\\)?:? #[0-9]+
+suppress_comment=\\(.\\|\n\\)*\\$FlowFixedInNextDeploy
+suppress_comment=\\(.\\|\n\\)*\\$FlowExpectedError
+
+unsafe.enable_getters_and_setters=true
+
+[version]
+^0.49.1
```
Comment (orta) [inline .flowconfig]: We shouldn't keep this file around, it'll confuse tooling

File: android/local.properties [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Thu Aug 24 18:06:49 EDT 2017
+sdk.dir=/Users/yuki/Library/Android/sdk
```
Comment (orta) [inline android/local.properties]: this'll need to be gitignored

File: index.android.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    margin: 10,
+  },
+  instructions: {
+    textAlign: 'center',
+    color: '#333333',
+    marginBottom: 5,
+  },
+});
+
+AppRegistry.registerComponent('android', () => android);
```
Comment (orta) [inline index.android.js]: Heh, that's a start

File: android/local.properties (hunk 2 above)
Comment (yuki24) [reply android/local.properties]: Oops, checked this in by accident.

File: .flowconfig (hunk 1 above)
Comment (yuki24) [reply .flowconfig]: 👍

**Comment by yuki24** [comment]
Actually... it was a lot easier to show the `Home` component than I'd imagined. So I added the commit that changed it. See the screenshot above. Images are still not showing up, but we can fix that very easily.

**Comment by ArtsyOpenSource** [comment]
## TSLint Issues:

* :no_entry_sign: <a href="https://github.com/artsy/emission/blob/android-second-try/src/lib/relay/config.android.tsx">config.android.tsx</a> - no-var-requires - require statement not part of an import statement

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by orta** [comment]
Sorry @yuki24 - this repo doesn't do prettier automatically for you, and prettier failed on CI, you can manually install the git hooks here - https://github.com/artsy/emission/blob/master/Makefile#L34-L36

**Comment by sarahscott** [comment]
🙀

**Comment by damassi** [comment]
@yuki24 / @orta  - should this maybe be closed until work resumes?

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/__tests__/OpaqueImageView.android-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "#skip_new_tests".
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  [danger-plugin-jest] Could not read test results. Danger cannot pass or fail the build.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by l2succes** [comment]
I'm gonna close this one until we give it another shot

**Pull request closed without merging**